walk/config.go

```diff
@@ -70,7 +83,7 @@ func (*Configurer) RegisterFlags(fs *flag.FlagSet, cmd string, c *config.Config)
 func (*Configurer) CheckFlags(fs *flag.FlagSet, c *config.Config) error { return nil }
 
 func (*Configurer) KnownDirectives() []string {
-	return []string{"exclude", "follow", "ignore"}
+	return []string{"generation_mode", "exclude", "follow", "ignore"}
```

Comment:
Please add docs for the new directive.

I don't have an idea, but would prefer a more descriptive term if we can find one. We already have a `build_file_mode`.